src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const initialState = [];
```

Comment:
- [ ] @Okechukwu-muokwugwo great work in your redux, you forgot to include the action type for checking your status, kindly ensure it follows the duck pattern, as this is a project requirement.
- [ ] ![image](https://user-images.githubusercontent.com/61408860/200666816-8ad09758-8e78-4f49-a4b7-15f27d0e69b8.png)